inc/functions/options.php

```diff
@@ -573,7 +573,7 @@ function rocket_check_key() {
 
 		if ( 'USER_BLACKLISTED' === $body ) {
```

Comment:
Can we deprecate this value and then swap it out with `USER_DENYLIST` instead?

Reply:
That's the one instance I was talking about in the OP, it's coming from here: https://github.com/wp-media/wp-rocket.me/blob/33b97937de54fd390c249b5785eee1fe717e1b60/valid_key.php#L41

So this needs to be updated before updating the plugin

Reply:
That sounds like something @iCaspar could take care of in the other repo. And then circle back to fix it here.

Reply:
Ok, sounds like a plan. @Tabrisrp , let's look at the other repo together briefly, and I'll get it started.

Reply:
we'll need to think of a way to preserve backward compatibility too on that one, as older versions of the plugin will expect the current response, while newer versions will expect the new one

Reply:
@GeekPress agreed that the version check we discussed would take care of back-compat issues. We actually have 2 instances in the wp-rocket.me repo that return that response. Besides the `valid_key.php` there is also one here: https://github.com/wp-media/wp-rocket.me/blob/master/check_update.php#L83

He's updated the code on the other repo and taken care of the inernal checks on redis storage, etc. Looks like a go for 3.6.2